requirements.txt

```diff
@@ -9,7 +9,7 @@ djangorestframework-simplejwt==4.7.2; python_version > '3.6'
 djangorestframework-xml==2.0.0
 djangorestframework-yaml==2.0.0
 PyJWT==2.1.0
-pymongo==3.12.0
+pymongo==3.7.2
```

Comment:
Was there an error with the newer version of pymongo?

Reply:
No

Reply:
I also tried MONGO_VERSION=5.0.3 and no problems. I know that is a breaking change with older data stored in mongo. I reverted back to your  MONGO_VERSION=4.4.6

Reply:
This is specific to pymongo. I will test on my end with both versions of pymongo and mongodb.

Reply:
Both versions of pymongo and mongodb have passed my testing. I am for bumping this to the latest version. For those that need to support version 4.x of MongoDB, that can be done by downgrading the version in dc_config/cybercom_config.env. Any objections to this?

Reply:
Agree, No Objections! I wanted to make sure your current volume would work with the new changes. I guess that is really easy to remove volume and `make init` again.